index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Reproductor de video FerTube</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<header>
+		<div class="logo">
+			<img src="./imagenes/Tube.png" alt="Logo FerTube">
+		</div>
+		<div class="search">
+			<input type="text" placeholder="Buscar...">
+			<button type="submit">Buscar</button>
+		</div>
+		<div class="menu">
+			<a href="#">Inicio</a>
+			<a href="#">Tendencias</a>
+			<a href="#">Historial</a>
+			<a href="#">Suscripciones</a>
+			<a href="#">Biblioteca</a>
+			<a href="#">Mi cuenta</a>
+		</div>
```

Comment:
It would be semantically convenient to place all the elements "a" inside an ordered list "li" and in turn, all these inside a tag "lu", since this would provide a clearer and more organized structure of the content, which would facilitate accessibility and website maintenance.